app/models/form/sales/questions/buyers_organisations.rb

```diff
@@ -15,4 +15,8 @@ def initialize(id, hsh, page)
     "pregla" => { "value" => "Local Authority" },
     "pregghb" => { "value" => "Help to Buy Agent" },
   }.freeze
+
+  def unanswered_error_message
+    "At least one option must be selected of these four"
+  end
```

Comment:
whoa. how is this working? I'm guessing there is some inherited behaviour from `::Form::Question`?
From playing in the app it looks to me like the default behaviour is to show "There is a problem \n You must answer #{@check_answer_label}" and this method just changes the second line of that. Is that right?

Reply:
hot stuff
last comment in case you are not aware of them (I am only just)
alternative syntax available:
`def unanswered_error_message = "At least one option ... "`
not sure it's better unless it's more widely used.